core/src/main/java/com/bankbazaar/tripmanager/model/Trip.java

```diff
@@ -32,17 +32,17 @@ public class Trip implements Serializable {
     private String tripDescription;
 
     @Column(name="start_date",nullable = false)
-    private Date startDate;
```

Comment:
Why change from Date?

Reply:
Date data type was saving time as 00:00:00 so i changed it to LocalDate. i have changed it back to Date